include/ScratchViews.h

```diff
@@ -571,7 +572,7 @@ void fill_pre_req_data(ElemDataRequests& dataNeeded,
 void fill_master_element_views(ElemDataRequests& dataNeeded,
```

Comment:
I still like the possible path forward to use unique names since it is the AssembleElemAlg that calls this method. Moreover, in the arg, it "knows" whether a face or element data is required. This can be a possible goal. In the meantime, this looks like a good step forward.

Reply:
I like the unique names path too. perhaps we could get together later and determine the lists of master-element calls that would belong in each function if we were to split things into unique names.

Reply:
Will do.. In the meantime, feel free to process the push. If you still do not have permissions, let me know and I can do it. @rcknaus is making progress on the exposed_grad_op and the plumbing. I think that I can convert a wall function algorithm to use the new meFC interface in not much time.